projects/dot-rules/src/lib/styles/rule-engine.scss

```diff
@@ -14,6 +14,8 @@ $foreground-active: #848484;
 $text-emphasis: #fc7e1c;
 $warn: #cb3927;
 $brand-primary-one: #209021;
+$gray: #7e7a86;
```

Comment:
Are this the same grays we have in the app variables?

Reply:
yes